Clean up jenkins on success

### Description

Jenkins is running out of space.

`npm ERR! nospc ENOSPC: no space left on device, open`

**Expected behavior:** 

Jenkins should regularly delete node_modules on success of the PR build.

**Actual behavior:** 

Jenkins does not delete node_modules on success

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -24,6 +24,9 @@ pipeline {
 	post {
 		success {
 			githubNotify context: 'continuous-integration/jenkins/lisky', description: 'The build passed.', status: 'SUCCESS'
+			dir('node_modules') {
+				deleteDir()
+			}
 		}
 		failure {
 			githubNotify context: 'continuous-integration/jenkins/lisky', description: 'The build failed.', status: 'FAILURE'

